src/cmd/util_test.go

```diff
@@ -0,0 +1,82 @@
+package cmd
+
+import (
+	"encoding/json"
+	"io"
+	"os"
+	"strings"
+
+	"gopkg.in/yaml.v2"
+)
+
+type Operation string
+
+const (
+	Assign   Operation = "assign"
+	Create   Operation = "create"
+	Delete   Operation = "delete"
+	Example  Operation = "example"
+	Get      Operation = "get"
+	List     Operation = "list"
+	Update   Operation = "update"
+	Unassign Operation = "unassign"
+)
+
+// execute any OpsLevel CLI command
+func execCmd(cmd Operation, resource string, inputs ...string) ([]byte, error) {
+	cliArgs := []string{string(cmd), resource}
+	cliArgs = append(cliArgs, inputs...)
+
+	r, oldStdout := redirectStdout()
+	defer r.Close()
+
+	rootCmd.SetArgs(cliArgs)
+	err := rootCmd.Execute()
+
+	output := captureOutput(r, oldStdout)
+	return output, err
+}
+
+// redirectStdout redirects os.Stdout to a pipe and returns the read and write ends of the pipe.
+func redirectStdout() (*os.File, *os.File) {
```

Comment:
Recommend that this change to returning `(*os.File, *os.File, error)` so that the error from `os.Pipe()` is handled.